manager/src/lib.rs

```diff
@@ -150,8 +138,27 @@ impl CronManager {
             available_balance: 0,
             staked_balance: 0,
             agent_fee: u128::from(GAS_BASE_FEE),
-            slot_granularity: SLOT_GRANULARITY
-        }
+            slot_granularity: SLOT_GRANULARITY,
+            agent_storage_usage: 0
+        };
+        this.measure_account_storage_usage();
+        this
+    }
+
+    /// Measure the storage an agent will take and need to provide
+    fn measure_account_storage_usage(&mut self) {
+        let initial_storage_usage = env::storage_usage();
+        // Create a temporary, dummy entry and measure the storage used.
+        let tmp_account_id = "a".repeat(64);
+        let tmp_agent = Agent {
+            payable_account_id: tmp_account_id.clone(),
+            balance: U128::from(0),
```

Comment:
@mikedotexe my idiot thoughts here -- do we care at all storage costs here?

Reply:
Not sure if I understand the question exactly, but I believe if it's typed to be a `u128` or `U128` that it would "reserve" that many bits. So if you're referring to `U128::from(0)` versus the max value of a u128, I don't believe this makes a difference.